lib/react.dart

```diff
@@ -146,7 +147,7 @@ abstract class Component {
   /// Initializes context
   _initContext(context) {
     this.context = new Map.from(context ?? const {});
-    this.nextContext = this.context;
+    this.nextContext = new Map.from(this.context ?? const {});
```

Comment:
It is confusing to read this since `nextContext` is `dynamic`, and yet, we can somehow safely pass it to the `Map.from` constructor, which only accepts a `Map`.

This deserves a comment (for both `context` and `nextContext`) explaining that the type was loosened from `Map` to `dynamic` for the purposes of being able to have `Component2 implements Component` - but that only "legacy" context APIs are supported in `Component` - which means it will still be a `Map` on the JS side of things.

Reply:
Sounds Good.